CONTRIBUTING.md

```diff
@@ -83,101 +100,91 @@ pytest -rs tests/integrations/flask/  # Replace "flask" with the specific integr
+- Every commit should follow the [Commit Message Format](https://develop.sentry.dev/commit-messages/#commit-message-format) convention.
+- CHANGELOG.md is updated automatically. No human intervention is necessary, but you might want to consider polishing the changelog by hand to make it more user friendly by grouping related things together, adding small code snippets and links to docs, etc.
 
-   - Think about the minimum versions supported, and test each version in a separate env in `tox.ini`.
+### Manual Process
 
-   - Create a new folder in `tests/integrations/`, with an `__init__` file that skips the entire suite if the package is not installed.
+- On GitHub in the `sentry-python` repository, go to "Actions" and select the "Release" workflow.
+- Click on "Run workflow" on the right side, make sure the `master` branch is selected.
+- Set the "Version to release" input field. Here you decide if it is a major, minor or patch release. (See "Versioning Policy" below)
+- Click "Run Workflow".
 
-3. Update package metadata.
+This will trigger [Craft](https://github.com/getsentry/craft) to prepare everything needed for a release. (For more information, see [craft prepare](https://github.com/getsentry/craft#craft-prepare-preparing-a-new-release).) At the end of this process a release issue is created in the [Publish](https://github.com/getsentry/publish) repository. (Example release issue: https://github.com/getsentry/publish/issues/815)
 
-   - We use `extras_require` in `setup.py` to communicate minimum version requirements for integrations. People can use this in combination with tools like Poetry or Pipenv to detect conflicts between our supported versions and their used versions programmatically.
+Now one of the persons with release privileges (most probably your engineering manager) will review this issue and then add the `accepted` label to the issue.
 
-     Do not set upper-bounds on version requirements as people are often faster in adopting new versions of a web framework than we are in adding them to the test matrix or our package metadata.
+There are always two persons involved in a release.
 
-4. Write the [docs](https://github.com/getsentry/sentry-docs). Answer the following questions:
+If you are in a hurry and the release should be out immediately, there is a Slack channel called `#proj-release-approval` where you can see your release issue and where you can ping people to please have a look immediately.
 
-   - What does your integration do? Split in two sections: Executive summary at top and exact behavior further down.
+When the release issue is labeled `accepted`, [Craft](https://github.com/getsentry/craft) is triggered again to publish the release to all the right platforms. (See [craft publish](https://github.com/getsentry/craft#craft-publish-publishing-the-release) for more information.) At the end of this process the release issue on GitHub will be closed and the release is completed! Congratulations!
```

Comment:
```suggestion
When the release issue is labelled `accepted`, [Craft](https://github.com/getsentry/craft) is triggered again to publish the release to all the right platforms. (See [craft publish](https://github.com/getsentry/craft#craft-publish-publishing-the-release) for more information.) At the end of this process the release issue on GitHub will be closed and the release is completed! Congratulations!
```

Reply:
Would not change this, "labeled" is the preferred spelling in US English